content/pages/2022/tickets.md

```diff
@@ -7,24 +7,23 @@ template: page
 slug: tickets_2022
 ---
 
-# Tickets for EuroSciPy
+# Tickets for EuroSciPy 2022
```

Comment:
I don't think this is worth changing anything in previous page

Reply:
That's actually one of the most important changes here: search engines take "tickets for euroscipy" h1 header, and they have it indexed and "well established", without this, we'll have some engines like Bing pointing out to 2022 for this search query for the next couple of months

Reply:
Sorry, I was not precise enough. I did not mean the title but the tense in the section below.,

Reply:
Oh, that one is for Bing Chat 😆 
If you ask it now about tickets, it will do the search, and then it will talk about EuroSciPy 2022 as if it will be held in the future and what are the prices 🤦 

welcome to the future, I guess, I could never imagine I'd be solving problems like this

Reply:
it doesn't affect the validity of the content and hardly people will go back in time to see EuroSciPy 22 tickets. Will resolve this.